### ISSUE TITLE
Reliability ethercalc.org/net

### ISSUE THREAD

**Issue by miclill** [audreyt/ethercalc#790]
First of all, thank you for creating this software. It is truly useful!

As can be seen in quite a few issues (#788 #766 #737 ...) the service (ethercalc.org/net) is quite unreliable.
I would thus like to suggest to disable the creation of new spreadsheets until these issues are sorted out.

**Comment by ocdtrekkie** [comment]
It is probably useful as a demo of the software. People probably shouldn't rely on a public instance of open source software to preserve their data long-term.

**Comment by eddyparkinson** [comment]
I would agree that is does not run 24*365 and if you need it running all the time then host on your own server and monitor it. It will need a restart from time to time. But it is a very high quality feature rich spreadsheet. It far better then other open source spreadsheets I have looked at.  And I have looked at a lot of other open source spreadsheets.
